stormworksguide.md

```diff
@@ -0,0 +1,48 @@
+# Stormworks Setup Guide
+
+## Linux Support
+
+The server requires Wine to run on Linux, Preferably Wine 8. Using a container avoids the need to install this dependency on the host, otherwise,  Wine is required to be installed on the host system to run the server. 
+
+## Player Permissions
+
+There are 4 settings for player permissions. Admins, Authorized, Blacklist, and Whitelist. 
```

Comment:
Perhaps, note that you have to use the commands at the bottom to adjust these in-game? The way it reads now, I was confused what I'd do with these until I finished the whole guide.